routes/auth.py

```
@@ -0,0 +1,241 @@
+    })
+
+
+@auth_bp.route('/Passport/v2/RefreshToken', methods=['POST'])
+def passport_refresh_token():
+    """刷新Token"""
+    data = request.get_json()
+    refresh_token = data.get('RefreshToken', '')
+    
+    try:
+        decrypted_refresh_token = decrypt_data(refresh_token)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)